MailjetApiClient/Models/MailjetMail.cs

```diff
@@ -9,6 +9,7 @@ public class MailjetMail
         public int TemplateId { get; set; }
         public Dictionary<string, string> Variables { get; set; } 
         public List<MailAttachmentFile> AttachmentFiles { get; set; } = new List<MailAttachmentFile>();
-        public List<User> UsersInCc { get; set; } 
+        public List<User> UsersInCc { get; set; };
```

Comment:
Delete the useless semicolon
```suggestion
        public List<User> UsersInCc { get; set; }
```